core/contracts/financial-templates/implementation/FeePayer.sol

```diff
@@ -94,7 +107,9 @@ abstract contract FeePayer is Testable {
     }
 
     /**
-     * @notice Pays UMA DVM final fees to the Store contract. This is a flat fee charged for each price request.
+     * @notice Pays UMA DVM final fees to the Store contract.
+     * @dev This is a flat fee charged for each price request.
+     * @param payer address of who is paying the fees.
      * @return totalPaid The amount of collateral that was paid to the Store.
      */
     function _payFinalFees(address payer) internal returns (FixedPoint.Unsigned memory totalPaid) {
```

Comment:
nit: this is an internal function -- should it be in the internal functions section below?